src/components/Derivatives/Derivatives.columns.js

```diff
@@ -33,7 +34,7 @@ export default function getColumns(props) {
     {
       id: 'price',
       name: 'column.priceDeriv',
-      width: COLUMN_WIDTHS.AMOUNT,
+      width: getColumnWidth('price', columnsWidth) || COLUMN_WIDTHS.AMOUNT,
```

Comment:
As an idea, instead of using or here, wouldnt be better to have the failed to default inside getColumnWidth function?